spring-boot/admin/src/test/java/eu/coatrack/admin/controllers/SpringSecurityTestConfig.java

```diff
@@ -53,9 +52,11 @@ public UserDetailsService userDetailsService() {
                 basicUser));
     }
 
+/*
```

Comment:
I would add a TODO with reference to #183. Otherwise it would be dead code.

Reply:
Done